### PR TITLE
feature : refactor permission base on EWM project

### PR DESCRIPTION
**Pull request by jalalifardemos**
Description: Remove Some Permissions and add to every projects base on disabled methods in EWM Permission

### Linked PRs (optional)
List other PRs that are somehow connected to this and explain the connection.

- https://github.com/demos-europe/demosplan-project-ewm/pull/88
- https://github.com/demos-europe/demosplan-project-bimschgsh/pull/54
- https://github.com/demos-europe/demosplan-project-blp/pull/67
- https://github.com/demos-europe/demosplan-project-bobhh/pull/69
- https://github.com/demos-europe/demosplan-project-bobsh/pull/48
- https://github.com/demos-europe/demosplan-project-diplanbau/pull/166
- https://github.com/demos-europe/demosplan-project-diplanbimschg/pull/10
- https://github.com/demos-europe/demosplan-project-diplanfest/pull/39
- https://github.com/demos-europe/demosplan-project-diplanrog/pull/53
- https://github.com/demos-europe/demosplan-project-planfestsh/pull/53
- https://github.com/demos-europe/demosplan-project-regio/pull/56
- https://github.com/demos-europe/demosplan-project-robobsh/pull/58
- https://github.com/demos-europe/demosplan-project-teilhabe/pull/45

### CODE REVIEW

**Review by bakridemos — CHANGES_REQUESTED** [review]

File: demosplan/DemosPlanCoreBundle/Permissions/Permissions.php [hunk 1]
```diff
@@ -114,9 +114,6 @@ class Permissions implements PermissionsInterface, PermissionEvaluatorInterface
      */
     private array $addonPermissionCollections = [];
 
-    /**
-     * @param array<non-empty-string, PermissionInitializerInterface> $addonPermissionInitializers
-     */
```
[the remark this answers is not in the document]
Comment (jalalifardemos) [reply demosplan/DemosPlanCoreBundle/Permissions/Permissions.php:119]: done in [47cdefb](https://github.com/demos-europe/demosplan-core/pull/1983/commits/47cdefb5cfa55de5e67251e5ac199e7a714626e7)

**Review by bakridemos — DISMISSED** [review]
lgtm, code review only. Test in process

**Comment by jalalifardemos** [comment]
fixed permissiontest in core [542ed1c](https://github.com/demos-europe/demosplan-core/pull/1983/commits/5f6ded0f715a3fe916b543241718a92e39f39442)

**Review by bakridemos — APPROVED** [review]
permissionsTest is adjusted. Lgtm, code review and tested too ( according to permissionsTests, all tests results are same as main)

**Pull request merged**